### PR TITLE
New gun/sword company Jarnsmiour

### PR DESCRIPTION
**Pull request by desminus**
## About The Pull Request
Adds in the new Gun Company  Jarnsmiour, dealing with all of your bladed needs and things to general stab people with.
Allows access to buy weapons that were either niche to find or generally fits the bill, all within the nice umbrella of guncargo's 
system.
The items being,  The bowie knife, Cargo Sabre, and survival Knife
## How This Contributes To The Skyrat Roleplay Experience

Gun cargo system can be seen more as a way to obtain weapons and the like  with the permits and the like, so here adding in weapons and the like that would also be lovely to regulate with that as well.
## Changelog

:cl:
add: Jarnsmiour company
add: Added items to their shop

/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
I appreciate you are new at this and still learning desminus but this is not how to make PRs. You can't just
delete the file and upload it
you can just edit it

File: modular_skyrat/modules/gun_cargo/code/company_datums.dm [hunk 1]
```diff
@@ -1,111 +1,121 @@
...
+	name = "Armament Dynamics Inc."
+	can_roundstart_pick = FALSE
+	company_flag = COMPANY_DYNAMICS
+	cost_change_lower = -5000
+	cost_change_upper = 6500
+	cost = 4000 //subsidized or smth
+
+/datum/gun_company/jarnsmiour
+	name = "Jarnsmiour Blacksteel Foundation"
+	cost = 100
```
Comment (tf-4) [inline modular_skyrat/modules/gun_cargo/code/company_datums.dm:115]: this is 30x cheaper than the next cheapest company. Make it like 2000

File: modular_skyrat/modules/knives/knives.dm [hunk 2]
```diff
@@ -35,7 +35,7 @@
 		))
 		
 
-/obj/item/storage/belt/bowiesheath/AltClick(mob/user)
+/obj/item/storage/belt/bowie_sheath/AltClick(mob/user)
```
Comment (tf-4) [inline modular_skyrat/modules/knives/knives.dm:38]: undocumented

File: modular_skyrat/modules/gun_cargo/code/armament_datums/jorns.dm [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+/datum/armament_entry/cargo_gun/jarns/knife
+	subcategory = ARMAMENT_SUBCATEGORY_CQC
+
+/datum/armament_entry/cargo_gun/jarns/knife/bowie
+	item_type = /obj/item/storage/belt/bowie_sheath
+	lower_cost = CARGO_CRATE_VALUE * 3
+	upper_cost = CARGO_CRATE_VALUE * 5
+
+
```
Comment (tf-4) [inline modular_skyrat/modules/gun_cargo/code/armament_datums/jorns.dm:15]:
```suggestion
```

**Comment by YakumoChen** [comment]
> desminus:master

I'm sorry tali but 
image

**Review by tf-4 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**